samples/telemetry/debug.json

```diff
@@ -19,5 +15,16 @@
   },
   "view": {
     "id": "623d50fd-75cf-4025-97d2-e51ff94171f6"
+  },
+  "action": {
+    "id": "ae3a5d82-cdd1-468d-9bc9-3aa9e54d953c"
+  },
+  "telemetry": {
+    "status": "debug",
+    "message": "Session inconsistencies detected",
+    "debug": {
+      "cookie": "foo",
+      "session": "bar"
+    }
```

Comment:
This is not declared in the schema. Is schema not up-to-date?

Reply:
This data is just to let us know that all data in the telemetry object is forwarded. Maybe you can add comment to make it clearer.